_docs/v2.23/setup/migrate.md

```diff
@@ -11,7 +11,7 @@ deprecated Python Installer to the new Terraform module.
 If you have any
 questions about this process, please contact us by
 [email](mailto:discuss@forsetisecurity.org) or on
-[Slack](https://forsetisecurity.slack.com/join/shared_invite/enQtNDIyMzg4Nzg1NjcxLTM1NTUzZmM2ODVmNzE5MWEwYzAwNjUxMjVkZjhmYWZiOGZjMjY3ZjllNDlkYjk1OGU4MTVhZGM4NzgyZjZhNTE).
+[Slack](https://join.slack.com/t/forsetisecurity/shared_invite/enQtOTM4NTkwMDcwMDA1LTk1ZDExYTExZTJlNjY3NjIwZmVhZmJkMjk3YzVhZmYwNGRmYmU0N2UzZDc2Njg4NmEwYWU4ODc3MWI3NjJkZTE).
```

Comment:
Instead of joining, how about a more generic link that doesn't expire?  like `http://forsetisecurity.slack.com`?

Reply:
This invite is set to never expire, so we technically should not run into this issue in the future. I think you need to be invited to the slack workspace in order to join (can only sign in), so not sure that link would work.

Reply:
Sounds good, glad it won't expire.